react-app/tsconfig.json

```diff
@@ -1,14 +1,15 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ESNext",
```

Comment:
is this field case insensitive

Reply:
i think so but let's align to lower case